cmd/root.go

```diff
@@ -189,7 +189,7 @@ func Run(c *cobra.Command, names []string) {
 	httpAPI := api.New(apiToken)
 
 	if enableUpdateAPI {
-		updateHandler := update.New(func() { runUpdatesWithNotifications(filter) }, updateLock)
+		updateHandler := update.New(func(tags []string) { runUpdatesWithNotifications(filters.FilterByImageTag(tags, filter)) }, updateLock)
```

Comment:
```suggestion
		updateHandler := update.New(func(images []string) { runUpdatesWithNotifications(filters.FilterByImageTag(images, filter)) }, updateLock)
```
We should be consistent, but perhaps even `imageTags` is the most opaque term to use...

Reply:
So I understand that you think it should be changed to `imageTags` everywhere, is that correct?

Reply:
I've spotted the difference between "image" and "tag"; I've changed the parameter/variable names to image.